src/cephadm/cephadmlib/logging.py

```diff
@@ -142,29 +157,85 @@ def format(self, record: Any) -> str:
 """
 
 
+_VERBOSE_HANDLERS = [
```

Comment:
for my info, what makes these handlers the "verbose" handlers? This seems to just be all the main handlers we expect to be using.

Reply:
These are the handlers that are impacted when the --verbose flag is set. I'm open to naming suggestions here. :-)

Reply:
I don't have any better ideas, so I think we can leave it unless someone does